releases/v2.0-beta.20180312.md

```diff
@@ -0,0 +1,135 @@
+- In in geographical map on the homepage of the Admin UI enterprise version, node components now link to **Note Details** page. [#23283][#23283]
+- Removed the **Keys Written per Second per Store** graph. [#23303][#23303]
+- Added the **Lead Transferee** field to the **Range Debug** page. [#23241][#23241]
+
+### Bug Fixes
+
+- Fixed a correctness bug where some `ORDER BY` queries would not return the correct results. [#23541][#23541]
+- The Admin UI no longer hangs after a node's network configuration has changed. [#23348][#23348]
+- The binary format for [`JSONB`](../v2.0/jsonb.html) values is now supported. [#23215][#23215]
+- A node now waits in an unready state for the length of time specified by the `server.shutdown.drain_wait` cluster setting before draining. This helps ensure that load balancers won't send client traffic to a node about to be drained. [#23319][#23319]
+- Fixed a panic when using `UPSERT ... RETURNING` with `UNION`. [#23317][#23317]
+- Prevented disruptions in performance when gracefully shutting down a node. [#23300][#23300]
+- Hardened the [cluster version upgrade](../v2.0/upgrade-cockroach-version.html) mechanism. Rapid upgrades through more than two versions could sometimes fail recoverably. [#23287][#23287]
+- Fixed a deadlock when tables are rapidly created or dropped. [#23288][#23288]
+- Fix a small memory leak in [`IMPORT`](../v2.0/import.html) CSV. [#23259][#23259]
+- Prevented a panic in DistSQL under certain error conditions. [#23201][#23201]
+- Added a [readiness endpoint](../v2.0/monitoring-and-alerting.html#health-ready-1) (`/health?ready=1`) for better integration with load balancers. [#23247][#23247]
+- Fixed a zero QPS scenario when gracefully shutting down a node. [#23246][#23246]
+- Secondary log files (e.g., the SQL execution log) are now periodically flushed to disk, in addition to the flush
+  that occurs naturally when single log files are full (`--log-file-max-size`) and when the process terminates gracefully. Log file rotation is also now properly active for these files. [#23231][#23231]
+- Previously, the `ranges` column in the `node status` command output only included ranges whose raft leader was on the node in question. It now includes the count of all ranges on the node, regardless of where the raft leader is. [#23180][#23180]
+- Fixed a panic caused by empty `COCKROACH_UPDATE_CHECK_URL` or `COCKROACH_USAGE_REPORT_URL` environment variables. [#23008][#23008]
+- Prevented stale reads caused by the system clock moving backwards while the `cockroach` process is not running. [#23122][#23122]
+- Corrected the handling of cases where a replica fails to retrieve the last processed timestamp of a queue. [#23127][#23127]
+- Fixed a bug where the liveness status would not always display correctly on the single-node page in the Admin UI. [#23193][#23193]
+- Fixed bug with showing descriptions on the **Jobs** page in the Admin UI. [#23256][#23256]
```

Comment:
s/Fixed bug with showing/Fixed a bug that showed incorrect descriptions...

Reply:
Done.